src/libpmemobj/palloc.h

```diff
@@ -57,6 +59,10 @@ struct palloc_heap {
 	struct pool_set *set;
 
 	void *base;
+
+	int alloc_pattern;
```

Comment:
just change this to int64_t, and remove the padding.
And if you are worried about structure layout, you shouldn't use `int` anyway.

Reply:
done